appletree/component.py

```diff
@@ -268,22 +268,22 @@ def dependencies_deduce(
         self,
         data_names: Union[List[str], Tuple[str]] = ["cs1", "cs2", "eff"],
         dependencies: Optional[List[Dict]] = None,
-        nodep_data_name: str = "batch_size",
+        nodep_data_names: Union[List[str], Tuple[str]] = "batch_size",
```

Comment:
`"batch_size"` -> `["batch_size"]`

Reply:
Fixed in https://github.com/XENONnT/appletree/pull/187/commits/3023d945eadf5487e1c02316f71608ef6124944b